Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/README.md

```diff
@@ -14,6 +14,7 @@ Query the Symantec Endpoint Protection Manager using the official REST API.
 ## Required Permissions
 The following role is required to use the Symantec Endpoint Protection API:
 - sysadmin
+Note: "Administrator" role is not enough, integration commands require "System Administrator" role. 
```

Comment:
```suggestion
Note: An **Administrator** role does not have enough permissions for this integration. A **System Administrator** role is required. 
```
but why not just:
```suggestion
- System Administrator
```
and what is the sysadmin role (1 line above)? can a user have both roles?